packages/cli/src/TestWebhooks.ts

```diff
@@ -261,7 +271,7 @@ export class TestWebhooks implements IWebhookManager {
 				}
 			}
 
-			delete this.registrations[key];
+			await this.registrations.deregister(key);
```

Comment:
When doing the deregistration, would it make sense to consider batching them all and issuing a single deletion? To prevent multiple hops? Especially if we decide to use a hash.

Reply:
Removed [here]([5c2d670](https://github.com/n8n-io/n8n/pull/8176/commits/5c2d670e47d8a50c2c3bd7ad1271de8d6a6ba066)) this call which was also happening in `deactivateWebhooks`. We can optimize further once we have Redis hashes.